staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -98,6 +98,10 @@ type KubeletConfiguration struct {
 	// Default: ""
 	// +optional
 	StaticPodPath string `json:"staticPodPath,omitempty"`
+	// podLogsDir is a custom root directory path kubelet will use to place pod's log files.
+	// Default: "/var/log/pods"
```

Comment:
let's add the same comment about the temp folder not being appropriate as a logs path

Reply:
Added